index.bs

```diff
@@ -4043,6 +4063,18 @@ The [=remote end steps=] with |session| and |command parameters| are:
          1. Let |result nodes| be [=locate nodes using accessibility attributes=]
             given |context nodes|, |selector|, and |maximum returned node count|.
 
+      <dt>|type| is the string "<code>container</code>"
+      <dd>
+
+         1. If |start nodes parameter| is not null,
```

Comment:
I think this, and the confusion over whether the navigable needs to be a child of the target navigable, points slightly in the direction of a new command being preferable here, but I won't block on that or anything :)